### PR TITLE
Spelling/grammar check, link fix.

### PR DESCRIPTION
**Pull request by CatRabbit499**
- Spelling corrections
- Grammar corrections
- Change FML Source link to 1.13-pre from 1.12.x
- Update MCP link
- Changed Maven link to HTTPS

### CODE REVIEW

**Comment by liach** [comment]
the branch, i am sure, will change once a runnable 1.13 is out.

**Comment by kierajreed** [comment]
you have an extra capital on Line 19, just saying

File: CREDITS.txt [hunk 1]
```diff
@@ -2,45 +2,48 @@ Minecraft Forge: Credits/Thank You
 
 Forge is a set of tools and modifications to the Minecraft base game code to assist 
 mod developers in creating new and exciting content. It has been in development for 
-several years now, but I would like to take this time thank a few people who have 
-helped it along it's way.
+several years now, so now I would like to take this time thank a few people who have
```
Comment (liach) [inline CREDITS.txt]: No need for `now`

File: CREDITS.txt [hunk 2]
```diff
@@ -2,45 +2,48 @@ Minecraft Forge: Credits/Thank You
...
 mod developers in creating new and exciting content. It has been in development for 
-several years now, but I would like to take this time thank a few people who have 
-helped it along it's way.
+several years now, so now I would like to take this time thank a few people who have 
+helped it along its way.
 
-First, the people who originally created the Forge projects way back in Minecraft 
-alpha. Eloraam of RedPower, and SpaceToad of Buildcraft, without their acceptiance 
-of me taking over the project, who knows what Minecraft modding would be today.
+First, thanks to the people who originally created the Forge projects way back in
```
Comment (liach) [inline CREDITS.txt:8]: No need for `to` before the names of people because you are enumerating `a few people`.

File: CREDITS.txt [hunk 3]
```diff
@@ -2,45 +2,48 @@ Minecraft Forge: Credits/Thank You
...
+Thanks to Mezz for stepping up as the issue and pull request manager. He Helps to
+keep me sane while guiding the community into creating better additions to Forge.
 
-Searge, Bspks, Fesh0r, ProfMobious, and all the rest over on the MCP team {of which 
-I am a part}. For creating some of the core tools needed to make Minecraft modding 
+Searge, Bspkrs, Fesh0r, ProfMobious, and all the rest over on the MCP team (of which 
+I am a part) for creating some of the core tools needed to make Minecraft modding 
 both possible, and as stable as can be.
   On that note, here is some specific information of the MCP data we use:
     * Minecraft Coder Pack (MCP) *
```
Comment (liach) [inline CREDITS.txt]: You should rename this to `Mod Coder Pack` since its name has changed.

File: CREDITS.txt [hunk 4]
```diff
@@ -51,15 +54,15 @@ This support has been dropped as of Minecraft release 1.7, as Risugami no longer
...
+http://keithschwarz.com/interesting/code/?dir=topological-sort
 
-It also contains code from the Maven project for performing versioned dependency
-resolution. http://maven.apache.org/
+It also contains code from the Maven project for performing versioned dependency resolution.
+https://maven.apache.org/
 
-It also contains a partial repackaging of the javaxdelta library from http://sourceforge.net/projects/javaxdelta/
-with credit to it's authors.
+It also contains a partial repackaging of the javaxdelta library from SourceForge with credit to it's authors.
```
Comment (liach) [inline CREDITS.txt:63]: `it's` -> `its`. Your grammar failed!

File: CREDITS.txt [hunk 5]
```diff
@@ -51,15 +54,15 @@ This support has been dropped as of Minecraft release 1.7, as Risugami no longer
...
+http://sourceforge.net/projects/javaxdelta/
 
-Forge Mod Loader downloads components from the Minecraft Coder Pack
-(http://mcp.ocean-labs.de/index.php/Main_Page) with kind permission from the MCP team.
+Forge Mod Loader downloads components from the Minecraft Coder Pack with kind permission from the MCP team.
```
Comment (liach) [inline CREDITS.txt:66]: `Mod Coder Pack`, too.

**Review by liach — CHANGES_REQUESTED** [review]
I do not think this is worthy enough to create a pull request, and I believe this pull request has a very slight chance to be merged as it is trivial. Moreover, this pull request has a few issues; it even introduces more grammatical errors.

**Pull request closed without merging**